packages/SwingSet/test/abandon-export.test.js

```diff
@@ -43,7 +49,7 @@ const assertSingleEntryLog = (t, log, type, details = {}) => {
   return entry;
 };
 
-const makeTestVat = async (t, kernel, name) => {
+const makeTestVat = async (t, kernel, name, kernelKeeper) => {
```

Comment:
It looks like we can drop the `t` parameter.
```suggestion
const makeTestVat = async (kernel, name, kernelKeeper) => {
```